features/src/rust/install.sh

```diff
@@ -145,6 +145,10 @@ if [ "${UPDATE_RC}" = "true" ]; then
     add_etc_profile_d_script rust "$(cat .bashrc | envsubst "${vars_%,}")";
 fi
 
+# Completion
+rustup completions bash | sudo tee /usr/share/bash-completion/completions/rustup > /dev/null
+rustup completions bash cargo | sudo tee /usr/share/bash-completion/completions/cargo > /dev/null
```

Comment:
Don't need to use `sudo` here since we're already root when building the container.
```suggestion
rustup completions bash | tee /usr/share/bash-completion/completions/rustup > /dev/null
rustup completions bash cargo | tee /usr/share/bash-completion/completions/cargo > /dev/null
```